docs/reference/indices/resolve-cluster.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[indices-resolve-cluster-api]]
+=== Resolve cluster API
+++++
+<titleabbrev>Resolve cluster</titleabbrev>
+++++
+
+Resolves the specified index expressions to return information about
+each cluster, including the local cluster, if included.
+
+This endpoint is useful before doing a <<modules-cross-cluster-search,{ccs}>> in
+order to determine which remote clusters should be included in a search.
+
+You use the same index expression with this endpoint as you would for cross-cluster
+search. Index and <<exclude-problematic-clusters,cluster exclusions>> are also supported
+with this endpoint.
+
+For each cluster in the index expression, information is returned about:
+
+1. whether the querying ("local") cluster is currently connected to the remote cluster
+2. whether the remote cluster is configured with `skip_unavailable` as `true` or `false`
+3. whether there are any indexes, aliases or data streams on that cluster that match
+   the index expression
```

Comment:
Is the list of indices dependent on my permissions, and if so, should we mention that here?

Reply:
Permissions issues will be returned as error messages - so they fit into the fourth bullet below. I've updated the docs to indicate that.